learn-CSS-by-Building-a-blog/blog53.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Home | Learn Basic CSS by building a Blog</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display:700|Source+Sans+Pro:300,400&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div id="main">
+        <header>
+            <nav>
+                <ul class="nav-links">
+                    <li><a href="#">Intro</a></li>
+                    <li><a href="#">Blog</a></li>
+                </ul>
+                <a href="#" id="logo" >
+                    <img src="https://d33wubrfki0l68.cloudfront.net/e66392bc2941ad564b6956321e778f29073cf585/4190b/img/glyph-alternative.svg" alt="freeCodeCamp logo">
+                </a>
+            </nav>
+        </header>
+        <section class="container">
+            <div class="blog">
+                <div class="title">
+                    <h1>Blog</h1>
+                    <div class="divider"></div>
+                </div>
+                <div class="article">
+                    <div>
+                        <div class="title-date">
+                            <h2>Article 1</h2>
+                            <p class="date"></p>
+                        </div>
+                        <div class="article-text">
+                            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. 
+                                Labore dolorum animi minus, facere quam, iusto molestiae 
+                                voluptate aut corporis dolor esse nulla odit culpa velit! 
+                                Perferendis temporibus earum magnam adipisci unde, numquam 
+                                esse ipsa deleniti? Voluptas numquam est deleniti? Placeat, 
+                                aliquam. Tempore porro recusandae laudantium aperiam veniam 
+                                harum consequuntur aliquid. Eveniet porro odio aliquam itaque 
+                                magni id aspernatur ad ullam maiores vero tempore dolores culpa 
+                                non quo et, ab esse quam unde, eaque iste. Accusantium, iure?
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </div>
+</body>
+</html>
+
+<!-- 
+    Add below the `p` element an `a` element with the `href` to "article1.html".
```

Comment:
Below the `p` element, add an `a` element with the `href` set equal to "article1.html".